InMobi/src/main/java/com/mopub/mobileads/InMobiGDPR.kt

```diff
@@ -0,0 +1,14 @@
+package com.mopub.mobileads
+
+import org.json.JSONObject
+
+object InMobiGDPR {
```

Comment:
Are we sure we want to go ahead with `object`? Because this would create a singleton object and all. And I'm not sure that is required because the following things should be set/get only in `UiThread`. And also check how these will be called if publisher calls it from Java class.

Atleast I can see this is how it will be called from Java class.
`InMobiGDPR.INSTANCE.getGDPRConsentDictionary()`, there are ways to avoid `INSTANCE` but just wanted to you double check if this is how it is going to be called or not.